Makefile

```diff
@@ -40,15 +40,22 @@ deps/capnproto:
 	@mkdir -p deps
 	git clone https://github.com/sandstorm-io/capnproto.git deps/capnproto
 
-SOURCES=$(shell find src/base src/os src/ekam -name '*.cpp' | \
+SOURCES=$(shell cd src; find base os ekam -name '*.cpp' | \
     grep -v KqueueEventManager | grep -v PollEventManager | \
     grep -v ProtoDashboard | grep -v ekam-client | grep -v _test)
 
-bin/ekam-bootstrap: $(SOURCES)
+OBJ_DIR := tmp
+OBJECTS=$(addprefix $(OBJ_DIR)/, $(SOURCES:.cpp=.o))
+
+$(OBJ_DIR)/%.o: src/%.cpp
+	@mkdir -p $(@D)
+	$(CXX) -Isrc -fPIC -std=c++14 -pthread -o $@ -c $<
```

Comment:
Since it's used in more than one place now, let's factor cflags out into a variable.